src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Tests.Button_IsEnabled_Automated.cs

```diff
@@ -256,5 +256,35 @@ void Check(bool expected, string msg)
 			// Assert after double click while myCheckBox is enabled
 			Check(true, "Assert after another double click while myCheckBox should stay checked");
 		}
+
+		[Test]
+		[AutoRetry]
+		public void Button_NestedButtons_Validation()
+		{
+			Run("nVentive.Umbrella.Views.UI.Samples.Content.UITests.ButtonTestsControl.Nested_Buttons");
+
+			var outer = _app.Marked("Outer");
+			var inner = _app.Marked("Inner");
+			var validateCommandTextBlock = _app.Marked("ValidateCommandTextBlock");
+
+			// Initial validation
+			Assert.AreEqual("", validateCommandTextBlock.GetDependencyPropertyValue("Text")?.ToString());
+
+			// Click on outer button and validate
+			_app.TapCoordinates(outer.FirstResult().Rect.X + 30, outer.FirstResult().Rect.Y + 30);
+			string command1 = validateCommandTextBlock.GetDependencyPropertyValue("Text")?.ToString();
+			if (command1 != null && !command1.Contains("OuterCommand"))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this call to 'command1.Contains' to an overload that accepts a 'StringComparison' as a parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4958644)